lib/Sender.js

```diff
@@ -315,10 +315,5 @@ function getArrayBuffer(data) {
 }
 
 function getRandomMask() {
-  return new Buffer([
-    ~~(Math.random() * 255),
-    ~~(Math.random() * 255),
-    ~~(Math.random() * 255),
-    ~~(Math.random() * 255)
-  ]);
+  return require('crypto').randomBytes(4);
```

Comment:
Only a tiny nit: would you mind moving the require at the top of the file along with the others?